autoload/neomake/makers/ft/vhdl.vim

```diff
@@ -0,0 +1,16 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#vhdl#EnabledMakers()
+    if executable('ghdl')
+        return ['ghdl']
+    else
+        return []
+    endif
```

Comment:
You should just return `['ghdl']` here - the executable-check gets done by Neomake.